src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,111 @@
+import "./footer.css";
+
+export default function Footer() {
+  return (
+    <footer className="footer" id="footer">
+      <div className="footer-columns">
+        <span className="col">
+          <h3 className="col-title">Contact</h3>
+          <nav className="col-list">
+            <ul>
+              <li>
+                <a href="./">hello@halcyonskinstore.com</a>
+              </li>
+              <li>
+                <a href="./">Chat with Us</a>
+              </li>
+              <li>
+                <a href="./">Call Us</a>
+              </li>
+              <li>Weekdays 09:00 - 17:00 Weekends 11:00 - 16:00</li>
+            </ul>
+          </nav>
+        </span>
+        <span className="col">
+          <h3 className="col-title">Explore</h3>
+          <nav className="col-list">
+            <ul>
+              <li>
+                <a href="./Product List/product.html">All Products</a>
+              </li>
+              <li>
+                <a href="./Cart/cart.html">Cart</a>
+              </li>
+              <li>
+                <a href="./Wishlist/wishlist.html">Wishlist</a>
+              </li>
+            </ul>
+          </nav>
+        </span>
+        <span className="col">
+          <h3 className="col-title">Info</h3>
+          <nav className="col-list">
+            <ul>
+              <li>
+                <a href="#">About Us</a>
```

Comment:
Don't use anchor tag instead use thr link component from the react-router-dom, as this will make your application a SPA

Reply:
Thanks Vinayak will work on these changes! :)